src/Nethermind/Nethermind.TxPool/ITxSender.cs

```diff
@@ -23,6 +23,6 @@ namespace Nethermind.TxPool
 {
     public interface ITxSender
     {
-        ValueTask<Keccak?> SendTransaction(Transaction tx, TxHandlingOptions txHandlingOptions);
+        ValueTask<(Keccak?, AddTxResult?)> SendTransaction(Transaction tx, TxHandlingOptions txHandlingOptions);
```

Comment:
Add names to tuple elements ValueTask<(Keccak? Hash, AddTxResult? AddResult)>, should they both be nullable?